content/en/getting_started/code_analysis/_index.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Getting Started with Code Analysis
+further_reading:
+- link: 'https://www.datadoghq.com/blog/datadog-code-analysis/'
+  tag: 'Blog'
+  text: 'Ship high-quality, secure code faster with Datadog Code Analysis'
+- link: 'https://www.datadoghq.com/blog/datadog-software-composition-analysis/'
+  tag: 'Blog'
+  text: 'Mitigate vulnerabilities from third-party libraries with Datadog Software Composition Analysis'
+- link: '/code_analysis/'
+  tag: 'Documentation'
+  text: 'Learn about Code Analysis'
+- link: '/security/application_security/software_composition_analysis'
+  tag: 'Documentation'
+  text: 'Learn about Software Composition Analysis'
+algolia:
+  tags: ["code analysis", "code analysis datadog", "code analysis ci pipeline", "code analysis ci pipelines"]
+---
+
+## Overview
+
+[Datadog Code Analysis][1] allows you to identify and resolve code quality issues and security vulnerabilities directly in Datadog or in your CI pipelines before deployment, ensuring a robust production build. 
+
+{{< img src="/code_analysis/repositories.png" alt="Session Replay available button, as well as visualization options" style="width:100%" >}}
+
+Code Analysis offers a comprehensive suite of tools, including [Static Analysis][2] and [Software Composition Analysis][3], to improve overall software delivery.
+
+* Static Analysis directly scans your repositories for bugs and performance issues, and suggests fixes to prevent these issues from impacting production. 
```

Comment:
Static Analysis (SAST) scans your repositories for quality and security issues in first-party code, and suggests fixes to prevent these issues from impacting production.

Reply:
```suggestion
* Static Analysis (SAST) scans your repositories for quality and security issues in first-party code, and suggests fixes to prevent these issues from impacting production.
```